src/main/java/com/_up/megastore/controllers/requests/UpdateSizeRequest.java

```diff
@@ -0,0 +1,14 @@
+package com._up.megastore.controllers.requests;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+public record UpdateSizeRequest (
+        @NotNull(message = "Size name must not be null")
```

Comment:
Esta etiqueta NotNull valida que si o si se envíe el nombre, pero puede enviarse una cadena vacío y lo toma como valido (el cual no es correcto).

Para esta validación extra existe una etiqueta llamada NotBlank el que además de verificar que se envíe (que no sea null) verifica que no sea una calida vacía.

```suggestion
        @NotBlank(message = "Size name must not be null")
```